### DISCUSSION TITLE
[ FF7 ] Crash on the native Spanish version

### DISCUSSION THREAD

**Discussion by ErickASJ1977** [julianxhokaxhiu/FFNx#374] [Q&A]
Greetings.
Is the mod spanish compatible?
I am very, very grateful for your great work. The game works perfectly. However, when I tried to install this mod over a fresh Spanish installation on Steam, the game crashes when invoked from the launcher. I get an "unhandled exception" message, followed by an invitation to check your FAQ.

I used version v1.10.1. I don't use any other mod. Tried changing API but had the same problem. According to the log, it wasn't able to locate some files.

If the mod does not support spanish, i would love to colaborate in some way. I can translate! :)

**Comment by julianxhokaxhiu** [comment]
Hi @ErickASJ1977,

Thanks for having me approached. I'd say that officially it does although it's also true we're working on so many things that it could be some of those feature are not yet compatible with the official spanish release.

Although what you can do in the meanwhile, is use this translation pack and you should be good to go anyway:
https://traduccionesxt.blogspot.com/2022/01/retraduccion-ffvii-version-pc-version.html
